lib/src/asset/exceptions.dart

```diff
@@ -11,3 +12,32 @@ class AssetNotFoundException implements Exception {
   @override
   String toString() => 'AssetNotFoundException: $assetId';
```

Comment:
Tried that out but the message doesn't seem very good? (all you get is the type)